analyzers/tests/SonarAnalyzer.UnitTest/PackagingTests/CsRuleTypeMapping.cs

```diff
@@ -1,7000 +0,0 @@
-﻿/*
- * SonarAnalyzer for .NET
- * Copyright (C) 2015-2022 SonarSource SA
- * mailto: contact AT sonarsource DOT com
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 3 of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with this program; if not, write to the Free Software Foundation,
- * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
- */
-
-namespace SonarAnalyzer.UnitTest.PackagingTests
-{
-    internal static class CsRuleTypeMapping
-    {
-        #region RuleTypesCs
-#pragma warning disable SA1005, SA1507 // SingleLineCommentsMustBeginWithSingleSpace, CodeMustNotContainMultipleBlankLinesInARow
-        // The empty lines below are so the rule number aligns nicely with the line number.
```

Comment:
The empty lines are practically not needed, we don't modify this manually. And were out of sync anyway.